test/cosmoz-tree_test.html

```diff
@@ -183,34 +220,33 @@
 				});
 			});
 
-			test('instantiating a Cosmoz.Tree works', function () {
-				assert.isNotNull(missingAncestorTree);
+			test('instantiating a Cosmoz.Tree', function () {
+				assert.isOk(missingAncestorTree);
 			});
 
 			test('getNodeByProperty', function () {
-				var node2 = missingAncestorTree.getNodeByProperty('id', '167d1485-7d4f-4c7d-86cd-a4fb00f31245'),
-					node301 = missingAncestorTree.getNodeByProperty('id', '3a7654f1-e3e6-49c7-b6a8-a4fb00f31245'),
-					node401 = missingAncestorTree.getNodeByProperty('id', '865065da-f44c-472e-a8df-a4fb00f3124b');
-				assert.isNotNull(node2);
-				assert.isNotNull(node301);
-				assert.isNotNull(node401);
+				var node2 = missingAncestorTree.getNodeByProperty('167d1485-7d4f-4c7d-86cd-a4fb00f31245', 'id'),
+					node301 = missingAncestorTree.getNodeByProperty('3a7654f1-e3e6-49c7-b6a8-a4fb00f31245', 'id'),
+					node401 = missingAncestorTree.getNodeByProperty('865065da-f44c-472e-a8df-a4fb00f3124b', 'id');
+				assert.isOk(node2);
+				assert.isOk(node301);
+				assert.isOk(node401);
 
 			});
 
-			test('getPath', function () {
+			test('getPathNodes', function () {
```

Comment:
@nomego please take a look at this test suite 'missingAncestor'.

Do we need to support this case? (Then getPathNodes() need to be re-written.)

See https://github.com/Neovici/cosmoz-tree/blob/master/test/data/missingAncestorTree.json